src/test/compile-fail/issue-39122.rs

```diff
@@ -8,6 +8,6 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-type Foo<T: std::ops::Add> = T; //~ WARNING E0122
+type Foo<T: std::ops::Add> = T; //~ WARNING bounds on generic type parameters are ignored
 
-type Bar<T> where T: std::ops::Add = T; //~ WARNING E0122
+type Bar<T> where T: std::ops::Add = T; //~ WARNING where clauses are ignored
```

Comment:
This test is now a strict subset of the new `ui/param-bounds-ignored`. Should I remove it?

Reply:
Yeah, the less code the better!